### DISCUSSION TITLE
training not coverging in han_imdb.py

### DISCUSSION THREAD

**Discussion by gamersover** [pyg-team/pytorch_geometric#4526] [Q&A]
I have ran examples/hetero/han_imdb.py, and the result looks not good.
![企业微信截图_1650875854377](https://user-images.githubusercontent.com/26238198/165052332-9a64144c-6127-4c5c-a73d-db6fe2bc1ecb.png)

**Comment by rusty1s** [comment]
@wsad1 Any thoughts?
